app/services/search_orders.rb

```diff
@@ -19,7 +19,7 @@ def fetch_orders
       includes(:payments, :subscription, :shipments, :bill_address, :distributor, :order_cycle).
       ransack(params[:q])
 
-    @search.result(distinct: true).joins(:bill_address)
+    @search.result.left_joins(:bill_address).uniq
```

Comment:
When I tried the following query, The query fails when it comes to sort the order by the billing address 
check `spec/controllers/api/v0/orders_controller_spec.rb:143`
```ruby
@search.result(distinct: true).left_joins(:bill_address)
```
and with this
```ruby
@search.result.left_joins(:bill_address).uniq
```
pagy fails on  `Api::V0::OrdersController` and multiple tests fail.
```ruby
...
        if pagination_required?
          @pagy, orders = pagy(orders,
                               items: params[:per_page] || default_per_page)
        end
...
```